pytest/lib/cluster.py

```diff
@@ -361,9 +363,9 @@ def cleanup(self):
         try:
             self.kill()
         except:
-            print("Kill failed on cleanup!")
+            logger.info("Kill failed on cleanup!")
             traceback.print_exc()
-            print("\n\n")
+            logger.info("\n\n")
```

Comment:
logger.critical('Kill failed on cleanup!', exc_info=sys.exc_info())

should do the trick, no?